resources/keb/templates/migrator-job.yaml

```diff
@@ -55,7 +59,7 @@ spec:
           {{- end }}
         {{- end}}
         - name: migrator
-          image: {{ .Values.global.images.container_registry.path }}/{{ .Values.global.images.schema_migrator.dir }}/schema-migrator:{{ .Values.global.images.schema_migrator.version }}
+          image: {{ .Values.global.images.container_registry.path }}/{{ .Values.global.images.schema_migrator.dir }}schema-migrator:{{ .Values.global.images.schema_migrator.version }}
```

Comment:
Values `schema_migrator.dir` does not have trailing slash. What is the reason for that change? If it is needed values.yaml should be changed accordingly.

Reply:
You are right. The problem is that the "dir" for CN must be empty. I've modified the main values yaml for that